dash/development/_py_components_generation.py

```diff
@@ -249,6 +251,8 @@ def create_docstring(component_name, props, description):
                 else prop['flowType'],
                 required=prop['required'],
                 description=prop['description'],
+                default=prop['defaultValue']['value']
+                if 'defaultValue' in prop.keys() else '',
```

Comment:
🐄 `'defaultValue' in prop` is sufficient, `.keys()` is unneeded

Reply:
this gets repeated a few times - 🌴 ?

Reply:
I'm a bit conflicted about moving this out into a function or something. It would need to be at the topmost scope, and only gets used three times. Is there another way to do this without repeating myself?

Reply:
(The `.keys()` have been removed in 9271f5b and f5bbab4 (I found another instance of it being used, so I went ahead and fixed that too.)

Reply:
I tried doing this, just for brevity: bbccf47

Reply:
I probably would have just made a top-level function but that's OK, we can leave it inline. I don't think bbccf47 is an improvement - shorter in one case, longer in the others, and less readable.

Though, the other thing you could do is move this logic into `create_prop_docstring` - ie pass the whole object in rather than the string value.

Reply:
The prop objects wouldn't have been trivial to extract in situations like this: https://github.com/plotly/dash/blob/cfa120ef4328a7895d7611020218b896d9ce29a3/dash/development/_py_components_generation.py#L245


So I compromised with 20cf393!